usecases/schema/manager.go

```diff
@@ -347,16 +348,25 @@ func (m *Manager) ResolveParentNodes(class, shardName string) (map[string]string
 	return name2Addr, nil
 }
 
-func (m *Manager) TenantShard(class, tenant string) (string, string) {
+func (m *Manager) TenantShard(class string, tenant string) (string, string) {
 	// TODO-RAFT: we always query the leader
 	// we need to make sure what is the side effect of
 	// if the leader says "tenant is HOT", but locally
 	// it's still COLD.
-	resTenant, resStatus, _, err := m.metaWriter.QueryTenantShard(class, tenant)
+	tenantsStatus, err := m.metaWriter.QueryTenantsShards(class, tenant)
 	if err != nil {
 		return "", ""
 	}
-	return resTenant, resStatus
+
+	return tenant, tenantsStatus[tenant]
+}
+
+func (m *Manager) TenantsShards(class string, tenants ...string) (map[string]string, error) {
```

Comment:
question: any reason to have this deduplication ? Is it just to avoid updating the interfaces/fakes/usage everywhere ?

Reply:
actually `TenantShard` should be removed, it's removed from the interface

Reply:
it should be already removed , that's weird, because it's not local and it's not staged 🤔